dapps/marketplace/src/pages/transaction/Transaction.js

```diff
@@ -207,25 +216,32 @@ require('react-styl')(`
       font-size: 14px
       font-weight: normal
     > h2
-      font-family: var(--heading-font)
-      font-size: 40px
-      font-weight: 200
+      font-family: var(--default-font)
+      font-size: 36px
+      font-weight: normal
       color: var(--dark)
       line-height: 3rem
     h3
       font-family: var(--heading-font)
-      font-weight: 300
-      font-size: 24px
-    .side-bar
-      h3
-        font-size: 18px
-        &.mt-4
-          margin-top: 1.5rem
+      font-weight: 500
+      font-size: 16px
+      color: var(--dark)
+      margin-bottom: 0.875rem
     .about-party
       margin-bottom: 2rem
     .help-icon
       margin-left: 0.5rem
       vertical-align: sub
+    .vertical-separator
+      height: 1px
+      background-color: #dde6ea
+    .horizontal-separator
+      flex: 0 0 13%
+      max-width: 13%
+      > div
```

Comment:
can this be an :after css class to avoid styling divs

Reply:
Ah yes good point tnanks